src/Factory/ClientFactory.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAI\Factory;
+
+use GuzzleHttp\Client as GuzzleClient;
+use OpenAI\Client;
+use OpenAI\Transporters\HttpTransporter;
+use OpenAI\ValueObjects\ApiKey;
+use OpenAI\ValueObjects\Transporter\BaseUri;
+use OpenAI\ValueObjects\Transporter\Headers;
+
+final class ClientFactory
```

Comment:
Can you add some PHP annotations - like we on the rest of the code to this class methods / properties.

Reply:
tbh, I don't know, what PHP annotations could bring any value here. Do you mean comments explaining the properties?

Reply:
Do you need any further information from me, @nunomaduro? :)